on-prem-docs/migration.md

```diff
@@ -0,0 +1,251 @@
+# Migrate to a Deployment of Automate Builder
+
+Long Term Support for On-Prem Builder is provided for installations of [Builder via Chef Automate](https://automate.chef.io/docs/on-prem-builder/).
+This document will guide you through the steps necessary to migrate your On-Prem Builder data into an Automate Builder.
+
+NOTE: The migration will overwrite any existing data in Automate Builder (target) with the data from your current On-Prem (source) installation. The assumption is that the Automate Builder target node is not yet in use.
```

Comment:
I think this might be worth calling out a bit stronger.  Perhaps making this section `Prerequisites`?

Reply:
Definitely! Handled in https://github.com/habitat-sh/on-prem-builder/pull/247/commits/161b40423ce5dfe30e11905fef68e37bc6ef894b